wp-content/plugins/core/src/Components/Components_Subscriber.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Components;
+
+use Tribe\Libs\Container\Abstract_Subscriber;
+use Tribe\Libs\Utils\Markup_Utils;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+/**
+ * Class Components_Subscriber
+ *
+ * @package Tribe\Project\Components
+ */
+class Components_Subscriber extends Abstract_Subscriber {
+
+	public function register(): void {
+		add_filter( 'tribe/libs/twig/funcitons', function ( $fs ) {
```

Comment:
`tribe/libs/twig/funcitons` Typo?

Reply:
It's actually a typo in base Gutenpanels, so I had to replicate it here to get it to fire. Once that's fixed in GP we can fix it here, too :)